activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   Fix `db:prepare` task to create/migrate test db when dev db already exists.
+
+    Fixes #44815.
+
+    *fatkodima*
```

Comment:
Can you remove the changelog entry. We don't usually mention bug fixes.

Reply:
Sure, but I'm seeing several bug fixes in the changelog - https://github.com/rails/rails/blob/main/activerecord/CHANGELOG.md ?

Reply:
Well, they weren't supposed to be added so let's not add one more that will need to be removed.

Reply:
👍 Removed.